gallery/src/components/NxFormSelect/NxFormSelectPage.tsx

```diff
@@ -109,6 +111,32 @@ const NxFormSelectPage = () =>
             </NxTable.Row>
           </NxTable.Body>
         </NxTable>
+        <NxH3>NxFormSelect CSS Classes</NxH3>
```

Comment:
Should be a separate tile subsection

Reply:
> Should be a separate tile subsection

I cleaned up the `NxTextInput` page as well: https://github.com/sonatype/sonatype-react-shared-components/pull/586/commits/02e695d0f1b35ad83d743c4ac1a3da727cda700b